parm/OM_config

```diff
@@ -9,30 +9,64 @@ export JOBSobsmon=${om_dir_root}/jobs
 source ${om_dir_root}/ush/detect_machine.sh
 export MACHINE_ID=`echo "${MACHINE_ID%%.*}"`
 
+module use ${HOMEobsmon}/modulefiles
+
 case ${MACHINE_ID} in
 
    hera)
       export SUB="sbatch"
       export SERVICE_PARTITION="service"
+      export CARTOPY_DATA_DIR="/scratch1/NCEPDEV/da/Edward.Safford/noscrub/python/envs/obs-mon/share/cartopy"
 
-      ptmp="/scratch2/NCEPDEV/stmp3/$USER"
-      stmp="/scratch2/NCEPDEV/stmp1/$USER"
+      ptmp="/scratch2/NCEPDEV/stmp3/${USER}"
+      stmp="/scratch2/NCEPDEV/stmp1/${USER}"
       queue=""
       project=""
       account="da-cpu"
       aprun_py="python"
+      module load obs-monitor/hera
```

Comment:
why not a single `module load obs-monitor/${MACHINE_ID}` line?

Reply:
Done.